### ISSUE TITLE
Better test naming convention?

### ISSUE THREAD

**Issue by derekjn** [pipelinedb/pipelinedb#641]
I think our naming convention for many .sql tests is becoming a bit crowded. For example, using the convention that most of our .sql tests use, this is what a test for the GIS aggregate `st_astwkbagg` might look like: `cqastwkbagg` or `cqswastwkbagg`. Yikes!

Instead of the `cq` prefix with no spacing, I'm thinking that the following convention is clearer:
- `cqavg` -> `cont_avg`
- `cqswavg` -> `cont_sw_avg`
- `cqastwkbagg` -> `cont_gis_astwkbagg`

Basically, `cont` instead of `cq` where appropriate, and separate things with underscores. We could even be more explicit by using the full `continuous`, but `cont` already has that meaning so it's probably obvious enough.

Any thoughts on this?
